README.md

```diff
@@ -1324,6 +1324,22 @@
     };
     ```
 
+  - Self invoking constructors to facilitate invoking constructors with or without ``new`` keyword.
+
+    ```javascript
+    function Jedi(options) {
+      options || (options = {});
+      if (!(this instanceof Jedi)) {
```

Comment:
This is not something one should get used to.

When you start using ES6 classes (which we are) calling class Jedi as `Jedi({name: 'Jedi'})` will throw an Error.